webroot/src/components/CollapseCaretButton/CollapseCaretButton.vue

```diff
@@ -12,8 +12,19 @@
         tabIndex="0"
         class="collapse-caret-button"
     >
-        <img v-if="isUp" class="caret-img" src="@assets/icons/up-caret.svg" alt="Collapse Button Logo" />
-        <img v-else class="caret-img" src="@assets/icons/down-caret.svg" alt="Expand Button Logo" />
+        <svg
+            v-if="isUp" width="24" height="14" viewBox="0 0 24 14" role="img"
+            fill="none" xmlns="http://www.w3.org/2000/svg">
+            <title>Collapse Button Logo</title>
+            <path d="M2 11.6342L11.9054 1.72852L21.8111 11.6342"
+            stroke-width="3" stroke-linecap="round" stroke-linejoin="round"/>
+        </svg>
+        <svg v-else width="24" height="14"
```

Comment:
One more thing: If you remove the hardcoded `width` & `height` attributes here and instead set `width` & `height` in your call-site CSS, then future components can also use this icon with their own width / height as needed.

It's really the `viewBox` that's the key from the SVG side. 

You can also remove the `xmlns` attribute if you want.